hound/deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: hound
+  labels:
+    app: hound
+spec:
+  replicas: 2
+  # selector defaults from template labels
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: gcsweb
+    spec:
+      terminationGracePeriodSeconds: 30
+      containers:
+        - name: hound
+          image: gcr.io/google_containers/hound:0.1.0
```

Comment:
most of our images are tagged `vX.Y.Z` now

Reply:
ack switching